docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -8,6 +8,65 @@ ifdef::notBuildingForSite[]
 include::{includedir}/menu.adoc[]
 endif::notBuildingForSite[]
 
+== Getting Ehcache
+
+=== From Maven
+
+Add the following to the <dependencies> section in your pom.xml:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.ehcache</groupId>
+    <artifactId>ehcache</artifactId>
+    <version>3.10.0</version>
+</dependency>
+----
+
+=== From Gradle
+
+Add the following to the dependencies section of your build.gradle:
+
+[source,groovy]
+----
+dependencies {
+  implementation 'org.ehcache:ehcache:3.10.0'
+}
+----
+
+ehcache-xml and ehcache modules requires jaxb dependency. So If you are using jdk-9+ version then you should make jaxb dependency available at runtime.
```

Comment:
I think from here onwards we could split in to two sections:
- JAXB Dependencies:
- Jakarta Variants

The JAXB dependencies section can talk about the fact that we depend on a JAXB runtime. This is a maven/gradle declared runtime dependency. It will normally appear automatically. The dependency uses a version range though, this means it will float, and can therefore change from build to build. This means if the build must be reproducible then the build system should declare dependency on an explicit version.

The Jakarta variant section should talk about how to depend on the Jakarta variants from both Maven and Gradle, and how that complicates the dependency situation for Maven (the dependencies are declared as optional).

Reply:
Accommodated the review comment.